### PR TITLE
add payments p2ms, p2pk, p2pkh, p2sh, p2wpkh, p2wsh

### PR DESCRIPTION
**Pull request by dcousens**
This will be a feature only PR,  with the removal of `templates` trialed/discussed in a different PR.

### CODE REVIEW

File: src/payments/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  p2pk: p2pk,
+  p2pkh: p2pkh,
+  p2sh: p2sh,
+  p2wpkh: p2wpkh,
+  p2wsh: p2wsh
+}
+
+// TODO
+// OP_RETURN
+// witness commitment
```
Comment (dcousens) [inline src/payments/index.js]: I'll leave this discussion to the `.templates` removal PR

**Comment by junderw** [comment]
cursory glance looks ok to me.

**Comment by junderw** [comment]
Node 6 and 7 are throwing invalid hex string....... hmmm

**Comment by dcousens** [comment]
Ready to merge! [if tests pass]

**Comment by dcousens** [comment]
@junderw merge if tests A-OK :+1:

**Pull request merged**